app/infrastructure/parsers/lms_parser.py

```
@@ -0,0 +1,301 @@
+            ):
+                assignment_list.append(activity)
+
+        if not activity_list:
+            return None
+
+        if assignment_list:
+            results = await asyncio.gather(
+                *(
+                    scrape_assignment_func(page_id=activity["id"])
+                    for activity in assignment_list
+                )
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)